app/flows/student_finance_calculator_flow/outcomes/_full_time_tuition.erb

```diff
@@ -2,18 +2,20 @@
 
 You could get a <%= format_money(tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/new-fulltime-students) each year to pay for your course.
 
-## Maintenance Loan
+<% if maintenance_loan_amount > 0 %>
```

Comment:
TODO: fill out commit message, and perhaps link to the commit/code where the logic is duplicated.